plugins/sip/lib/model/enums/HttpMethods.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @package plugins.sip
+ * @subpackage model.enum
+ */
+interface HttpMethods extends BaseEnum
+{
```

Comment:
Could not find usage of this class, and if it is used, need to move it to more generic location since it is valid for all and not just sip.

Reply:
Move to infra